### PR TITLE
feat: import protocols from protocols.io csv

### PR DESCRIPTION
**Pull request by fampinheiro**

### CODE REVIEW

**Comment by johnytiago** [comment]
I think you need to rebase this branch again. ff8ba68 is already on master

File: apps/asap-cli/src/cli.ts [hunk 1]
```diff
@@ -6,20 +6,27 @@
...
+    command: 'import <entity> <path>',
+    describe: 'import entities data to squidex from csv',
     builder: (cli) =>
-      cli.positional('path', {
-        describe: 'path to csv file',
-        type: 'string',
-      }),
-    handler: async ({ path }) => importUsers(path as string),
+      cli
+        .positional('entity', {
```
Comment (johnytiago) [inline apps/asap-cli/src/cli.ts:19]: Is entity required? Maybe we should use something like this http://yargs.js.org/docs/#demand

File: apps/asap-cli/src/import/protocols/insert.ts [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    },
+    tags: {
+      iv: data.keywords,
+    },
+  };
+
+  const [e1, teamId] = await Intercept(fetchTeamByName(data.team, promises));
+  const [e2, current] = await Intercept(fetchProtocolByLink(data.link));
+
+  const err1 = e1 as HTTPError;
```
Comment (johnytiago) [inline apps/asap-cli/src/import/protocols/insert.ts:112]: Isn't this just a got error?

File: apps/asap-cli/src/cli.ts (hunk 1 above)
Comment (fampinheiro) [reply apps/asap-cli/src/cli.ts:19]: The `<>` wrap ensures the property is required.

**Review by johnytiago — APPROVED** [review]

**Pull request merged**